srcpkgs/dmd2.091/template

```diff
@@ -0,0 +1,78 @@
+# Template file for 'dmd2.091'
+pkgname=dmd2.091
+version=2.091.0
+revision=1
+create_wrksrc=yes
+hostmakedepends="which"
+makedepends="dmd2.081 git"
+depends="gcc"
+short_desc="Digital Mars D compiler, 2.091"
+maintainer="Auri <me@aurieh.me>"
+license="BSL-1.0"
+homepage="http://www.digitalmars.com/d/2.0/"
+distfiles="
+ https://github.com/dlang/dmd/archive/v${version}.tar.gz>dmd-${version}.tar.gz
+ https://github.com/dlang/druntime/archive/v${version}.tar.gz>druntime-${version}.tar.gz
+ https://github.com/dlang/phobos/archive/v${version}.tar.gz>phobos-${version}.tar.gz
+ http://downloads.dlang.org/releases/2.x/${version}/dmd.${version}.linux.tar.xz"
+checksum="bcca38f4b80b51ae0a1955dc29e6cbfaa4d01d94869ddfcacf9292898c34343a
+ cc8ba196b08cca04488d97d0e35c878f7351b7d8128aec90eaa312d2bd9a6af2
+ df9f81eecda4366adc5ca9ed31a0b4474c248544b120ff6fa3f6cb45917d8769
+ 6e4bd4ee73a553921cdca7e3e21768b842b18186d07c0b4898fbd13a359875c0"
+conf_files="/etc/dmd.conf"
+provides="d-compiler-${version}_${revision}"
+conflicts="dmd-bootstrap dmd2.081 libphobos2.081"
+nopie=yes
+LDFLAGS="-lpthread"
+
+case "$XBPS_TARGET_MACHINE" in
+	x86_64*) _archbits=64;;
+	i686) _archbits=32;;
```

Comment:
No issue with using `i686*`, unless i686-musl is also unsupported by upstream.

Reply:
I do not believe i686-musl is supported upstream. I couldn't get it to build last I tried. I'll have another go with the new bootstrap chain.

Reply:
Tried building on `i686-musl`, `dmd2.081` fails because its druntime is missing `pthread_condattr_t` even though it's defined for musl in https://github.com/dlang/druntime/blob/fa3eeed29b5d892da921c585e6526635475059a4/src/core/sys/posix/sys/types.d#L852 *and* the version block is evaluated.

Reply:
Wow, that sucks. Thanks for testing!